layouts/ucf-faq-topic-card.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+if ( ! function_exists( 'ucf_faq_topic_list_display_card_before' ) ) {
+	function ucf_faq_topic_list_display_card_before( $content, $items, $args ) {
+		ob_start();
+	?>
+		<div class="faq-topic-list-wrapper">
+	<?php
+		return ob_get_clean();
+	}
+	add_filter( 'ucf_faq_topic_list_display_card_before', 'ucf_faq_topic_list_display_card_before', 10, 3 );
+}
+
+if ( ! function_exists( 'ucf_faq_topic_list_display_card_title' ) ) {
+	function ucf_faq_topic_list_display_card_title( $content, $items, $args ) {
+		$formatted_title = '';
+		if ( $title = $args['title'] ) {
+			$formatted_title = '<h2 class="ucf-faq-topic-list-title">' . $title . '</h2>';
+		}
+		return $formatted_title;
+	}
+	add_filter( 'ucf_faq_topic_list_display_card_title', 'ucf_faq_topic_list_display_card_title', 10, 3 );
+}
+
+if ( ! function_exists( 'ucf_faq_topic_list_display_card' ) ) {
+	function ucf_faq_topic_list_display_card( $content, $items, $args ) {
+		ob_start();
+		if( $items ):
+	?>
+			<div class="row">
+	<?php
+			foreach( $items as $key => $item ) :
+	?>
+				<div class="col-md-4 mb-4">
+					<div class="card h-100">
+						<div class="card-block">
+							<a href="<?php echo get_term_link( $item->term_id ); ?>">
+								<<?php echo $args['topic_element']; ?> class="ucf-faq-topic-title card-title <?php echo $args['topic_class']; ?>">
```

Comment:
What do you think about putting a `mb-0` class here so that the card's headline/title doesn't add additional margin to the bottom of the box, since the `.card-block` already provides 1.25rem of padding on the inside of the card.

**Without the class (with the bottom margin):**
![screen shot 2018-08-23 at 5 05 16 pm](https://user-images.githubusercontent.com/21267784/44551988-c6aa1100-a6f6-11e8-86cb-608bd6b37fef.png)

**With the class (without the bottom margin):**
![screen shot 2018-08-23 at 5 05 00 pm](https://user-images.githubusercontent.com/21267784/44552032-e6d9d000-a6f6-11e8-9549-0ed2bad4414b.png)